properties.schema

```diff
@@ -88,6 +88,15 @@
                   "validators": [],
                   "help": "Your default options here are: icon-globe, icon-language-1, icon-language-2"
                 },
+                "_restoreStateOnLanguageChange": {
+                  "type": "boolean",
+                  "required": true,
+                  "default": true,
```

Comment:
I think the default should be `false` to maintain backwards compatibility with existing courses.